platform/darwin/src/MGLStyleLayer.mm.ejs

```diff
@@ -14,6 +14,8 @@
 #import "MGLStyleValue_Private.h"
 #import "MGL<%- camelize(type) %>StyleLayer.h"
 
+#import "NSDate+MGLAdditions.h"
+
```

Comment:
Put this further up with the others, next to `#import "NSPredicate+MGLAdditions.h"`, a similar category use.